buildSrc/src/main/groovy/com/uber/okbuck/core/model/android/AndroidLibInstrumentationTarget.groovy

```diff
@@ -0,0 +1,41 @@
+package com.uber.okbuck.core.model.android
+
+import com.android.build.gradle.api.BaseVariant
+import com.android.build.gradle.api.LibraryVariant
+import com.android.builder.model.SourceProvider
+import org.gradle.api.Project
+
+/**
+ * An Android library instrumentation target
+ */
+class AndroidLibInstrumentationTarget extends AndroidAppInstrumentationTarget {
```

Comment:
This is a bit counterintuitive.

The app target extending the lib target makes more sense

Reply:
We should make a base `InstrumentationTarget` and add overrides in app and lib target classes